lib/components/eureka_rounded_button.dart

```diff
@@ -8,14 +8,16 @@ class EurekaRoundedButton extends StatefulWidget {
   final String buttonText;
   final Color buttonColor;
   final Color textColor;
+  final bool isTwoButtons;
 
   /// Users will need to pass in values for the onPressed interation and
   /// what the text for this button will be.
```

Comment:
Remove comments if they are not necessary.

Reply:
I think these kinds of comments useful

Reply:
For sure.